Library/Homebrew/formulary.rb

```diff
@@ -57,6 +57,7 @@ def self.class_s(name)
     class_name = name.capitalize
     class_name.gsub!(/[-_.\s]([a-zA-Z0-9])/) { $1.upcase }
     class_name.tr!("+", "x")
+    class_name.gsub!(/\b[@]\b/, "AT")
```

Comment:
Don't think you need the `[]`. Also, maybe can go for `/.@\d` to enforce a digit (for now at least, we can tweak it later). After that 👍 to shipping this.